pkg/kapp/resourcesmisc/custom_waiting_resource.go

```diff
@@ -96,6 +98,12 @@ func (s CustomWaitingResource) IsDoneApplying() DoneApplyState {
 					continue
 				}
 				if condMatcher.Success {
+					if condMatcher.SupportsUnblockingChanges {
```

Comment:
hmm, i just reread this section and i'm actually stuck now on why we are "unblocking" from the success block. if it's successful then it would be considered as done (and if it's done, it's unblocked). do we need to have non-success/non-failure condition that only allows to unblock?

Reply:
(e.g. in ytt, i could say im not "done", but i will unblock anyway)